docs/en/trust.rst

```diff
@@ -381,39 +362,48 @@ giving the references of the metadata protocol for each of these.
     
     The entries that don't have any reference to a known draft or standard are intended to be defined in this technical reference.
 
-+------------------+-----------------------------+--------------+
-| Entity           | metadata type               | references   |
-+==================+=============================+==============+
-| Trust Anchor     | ``federation_entity``       | `OIDC-FED`_  |
-+------------------+-----------------------------+--------------+
-| Intermediate     | ``federation_entity``       | `OIDC-FED`_  |
-+------------------+-----------------------------+--------------+
-|                  |                             |              |
-| Wallet Provider  | ``federation_entity``       | --           |
-|                  |                             |              |
-|                  | ``wallet_provider``         |              |
-|                  |                             |              |
-|                  |                             |              |
-+------------------+-----------------------------+--------------+
-|                  |                             |              |
-| Credential Issuer| ``federation_entity``       |              |
-|                  |                             |              |
-|                  | ``openid_credential_issuer``| `OPENID4VCI`_|
-+------------------+-----------------------------+--------------+
-|                  |                             |              |
-| Relying Party    | ``federation_entity``       |              |
-|                  |                             |              |
-|                  | ``wallet_relying_party``    | `OIDC-FED`_  |
-|                  |                             |              |
-|                  |                             | `OpenID4VP`_ |
-+------------------+-----------------------------+--------------+
+.. list-table::
+   :widths: 20 20 60
+   :header-rows: 1
+
+   * - Entity
+     - Metadata type
+     - References
+   * - Trust Anchor
+     - ``federation_entity``
+     - `OIDC-FED`_
+   * - Intermediate
+     - ``federation_entity``
+     - `OIDC-FED`_
+   * - Wallet Provider
+     - ``federation_entity``, ``wallet_provider``
+     - --
+   * - Authorization Server
+     - ``federation_entity``, ``oauth_authorization_server``
+     - `OPENID4VCI`_
+   * - Credential Issuer
+     - ``federation_entity``, ``openid_credential_issuer``, [``oauth_authorization_server``]
+     - `OPENID4VCI`_
+   * - Relying Party
+     - ``federation_entity``, ``wallet_relying_party``
+     - `OIDC-FED`_, `OpenID4VP`_
+
```

Comment:
I suggest to clarify here: we should distinguish between Entities, Roles, and Metadata Types. For example, the PID/(Q)EAA Provider is the Entity acting the roles of Authorization Server, Credential Issuer, and, optionally, Relying Party (Wallet or OpenID).